settings.gradle

```diff
@@ -31,4 +32,7 @@ include 'common',
         'test:testcommon',
         'test:integration',
         'test:system',
-        'bindings'
\ No newline at end of file
+        'bindings'
+include 'password-authplugin'
```

Comment:
what is password-authplugin?
also since its shared:basic-authplugin, what is this include for basic-authplugin?

Reply:
Thanks for catching those leftover lines. The IDE had added them when I was experimenting with adding Gradle modules and I forgot to remove them. I've now removed them in the latest commit that I just pushed.